### PR TITLE
Added File Enrichment - VMRay playbook

### PR DESCRIPTION
**Pull request by fseele-vmray**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Adds the "File Enrichment - VMRay" playbook.

## Screenshots
File_Enrichment_-_VMRay_Thu_Aug_19_2021

## Minimum version of Cortex XSOAR
- [X] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

**Review by barchen1 — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Please go over these notes:

1. Add to the README.md file a note regarding the change of the support for this pack.
You can take a look here: https://xsoar.pan.dev/docs/partners/adopt#process
2. Please add the `integrationReliability` parameter and output to the yml file, you can take a look at `Alexa Rank Indicator` as an example.

Let me know if you need help
Thanks

File: Packs/VMRay/Integrations/VMRay/VMRay.yml [hunk 1]
```diff
@@ -399,6 +399,33 @@ script:
...
+      type: String
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: The vendor used to calculate the score.
+      type: String
+    - contextPath: DBotScore.Score
+      description: The actual score.
+      type: Number
```
Comment (barchen1) [inline Packs/VMRay/Integrations/VMRay/VMRay.yml:428]: Please add the `Reliability` output.
You can take a look here - https://xsoar.pan.dev/docs/integrations/dbot#reliability-level, https://xsoar.pan.dev/docs/integrations/context-standards-mandatory

**Review by barchen1 — CHANGES_REQUESTED** [review]

**Comment by fseele-vmray** [comment]
Thank you for the feedback.

> 1. Add to the README.md file a note regarding the change of the support for this pack.

Not sure if I understand you correctly here. We already started the adoption process in a previous PR and the readme contains the mentioned message: https://github.com/demisto/content/blob/master/Packs/VMRay/README.md

File: Packs/VMRay/Integrations/VMRay/VMRay.py [hunk 2]
```diff
@@ -671,18 +671,33 @@ def get_sample_by_hash_command():
...
 
+        # DBotScore output
         scores = list()  # type: list
         for sample in entry_context[context_key]:
             scores += dbot_score_by_hash(sample)
         entry_context[outputPaths['dbotscore']] = scores
 
+        # Indicator output
+        # just use the first sample that is returned by the API for now
+        entry = entry_context[context_key][0]
```
[the remark this answers is not in the document]
Comment (fseele-vmray) [reply Packs/VMRay/Integrations/VMRay/VMRay.py:692]: It is possible (although very uncommon) to submit the same file (ie. same hash) in different ways to the VMRay sandbox which will result two different "sample" entities. But AFAICT, XSOAR is different in this regard - a File is uniquely identified by its hash. Therefore we just take the first sample entity returned by the API and return it as the Indicator.

**Review by barchen1 — CHANGES_REQUESTED** [review]
Please change the RN file from `1_1_1.md` to `1_1_2.md`, cause `1_1_1.md` is already exist.
Change in the `pack_metadata` file accordingly.

File: Packs/Palo_Alto_Networks_WildFire/Playbooks/playbook-Detonate_File_-_WildFire.yml [hunk 3]
```diff
@@ -479,102 +495,108 @@ inputs:
...
-  description: The type of the submission
+  description: The type of the submission.
   type: string
 - contextPath: WildFire.Report.Size
-  description: The size of the submission
+  description: The size of the submission.
   type: number
 tests:
-- Wildfire Test
+- No tests (auto formatted)
```
Comment (evisochek) [inline Packs/Palo_Alto_Networks_WildFire/Playbooks/playbook-Detonate_File_-_WildFire.yml:601]: Please update here the name of the test playbook

File: Packs/VMRay/TestPlaybooks/playbook-VMRay-test-File-Enrichment.yml [hunk 4]
```diff
@@ -0,0 +1,530 @@
...
+    skipunavailable: false
+    quietmode: 0
+  "24":
+    id: "24"
+    taskid: 9b100dcd-026d-4014-8eaf-74afff2085b1
+    type: playbook
+    task:
+      id: 9b100dcd-026d-4014-8eaf-74afff2085b1
+      version: -1
+      name: Invalid SHA256
```
Comment (evisochek) [inline Packs/VMRay/TestPlaybooks/playbook-VMRay-test-File-Enrichment.yml:475]: Please add the condition that the playbook won't stop on errors, as we discussed

File: Packs/VMRay/TestPlaybooks/playbook-VMRay-test-File-Enrichment.yml [hunk 5]
```diff
@@ -0,0 +1,530 @@
...
+    skipunavailable: false
+    quietmode: 0
+  "15":
+    id: "15"
+    taskid: 5a0f5737-e7ef-41de-808c-5abc69c5ca82
+    type: playbook
+    task:
+      id: 5a0f5737-e7ef-41de-808c-5abc69c5ca82
+      version: -1
+      name: Invalid MD5
```
Comment (evisochek) [inline Packs/VMRay/TestPlaybooks/playbook-VMRay-test-File-Enrichment.yml:146]: Please add the condition that the playbook won't stop on errors, as we discussed

File: Packs/VMRay/TestPlaybooks/playbook-VMRay-test-File-Enrichment.yml [hunk 6]
```diff
@@ -0,0 +1,530 @@
...
+    skipunavailable: false
+    quietmode: 0
+  "21":
+    id: "21"
+    taskid: cc61427b-41ef-4924-88fe-4c4097ccede7
+    type: playbook
+    task:
+      id: cc61427b-41ef-4924-88fe-4c4097ccede7
+      version: -1
+      name: Invalid SHA1
```
Comment (evisochek) [inline Packs/VMRay/TestPlaybooks/playbook-VMRay-test-File-Enrichment.yml:334]: Please add the condition that the playbook won't stop on errors, as we discussed

File: Packs/VMRay/TestPlaybooks/playbook-VMRay-test-File-Enrichment.yml (hunk 4 above)
Comment (fseele-vmray) [reply Packs/VMRay/TestPlaybooks/playbook-VMRay-test-File-Enrichment.yml:443]: Added the `continueonerror` property in f1489203a22035

Comment (fseele-vmray) [reply Packs/VMRay/TestPlaybooks/playbook-VMRay-test-File-Enrichment.yml:443]: The test playbook succeeds now:
image

**Review by evisochek — APPROVED** [review]

**Review by barchen1 — APPROVED** [review]

**Pull request merged**